app/src/main/kotlin/com/simplemobiletools/flashlight/screens/MainScreen.kt

```diff
@@ -0,0 +1,281 @@
+            verticalArrangement = Arrangement.SpaceEvenly
+        ) {
+            flashlightButton()
+            brightDisplayButton()
+            sosButton()
+            stroboscopeButton()
+            slidersSection()
+        }
+
+        Box(
+            modifier = Modifier.align(Alignment.BottomEnd),
+        ) {
+            sleepTimer()
+        }
+    }
+}
+
+@Composable
+internal fun FlashlightButton(
+    flashlightActive: Boolean,
+    onFlashlightPress: () -> Unit,
+) {
+    Icon(
+        modifier = Modifier
+            .size(AppDimensions.mainButtonSize)
+            .padding(vertical = Dimens.margin.normal)
+            .clickable(
+                indication = null,
+                interactionSource = rememberMutableInteractionSource(),
+                onClick = onFlashlightPress
+            ),
+        painter = painterResource(id = R.drawable.ic_flashlight_vector),
+        contentDescription = stringResource(id = R.string.flashlight_short),
+        tint = if (flashlightActive) MaterialTheme.colorScheme.primary else MaterialTheme.colorScheme.onSurface
+    )
+}
+
+@Composable
+internal fun BrightDisplayButton(
+    onBrightDisplayPress: () -> Unit,
+) {
+    Icon(
+        modifier = Modifier
+            .size(AppDimensions.smallerButtonSize)
+            .padding(vertical = Dimens.margin.normal)
+            .clickable(
+                indication = null,
+                interactionSource = rememberMutableInteractionSource(),
+                onClick = onBrightDisplayPress
+            ),
+        painter = painterResource(id = R.drawable.ic_bright_display_vector),
+        contentDescription = stringResource(id = R.string.bright_display),
+        tint = MaterialTheme.colorScheme.onSurface
+    )
+}
+
+@Composable
+internal fun SosButton(
+    sosActive: Boolean,
+    onSosButtonPress: () -> Unit,
+) {
+    Text(
+        modifier = Modifier
+            .padding(vertical = Dimens.margin.normal)
+            .clickable(
+                indication = null,
+                interactionSource = rememberMutableInteractionSource(),
+                onClick = onSosButtonPress
+            ),
+        text = stringResource(id = R.string.sos),
+        fontSize = AppDimensions.sosTextSize,
+        fontWeight = FontWeight.Bold,
+        color = if (sosActive) MaterialTheme.colorScheme.primary else MaterialTheme.colorScheme.onSurface
+    )
+}
+
+@Composable
+internal fun StroboscopeButton(
+    stroboscopeActive: Boolean,
+    onStroboscopeButtonPress: () -> Unit,
+) {
+    Icon(
+        modifier = Modifier
+            .size(AppDimensions.smallerButtonSize)
+            .padding(vertical = Dimens.margin.normal)
+            .clickable(
+                indication = null,
+                interactionSource = rememberMutableInteractionSource(),
+                onClick = onStroboscopeButtonPress
+            ),
+        painter = painterResource(id = R.drawable.ic_stroboscope_vector),
+        contentDescription = "",
```

Comment:
it better be null than empty, or just provide a string for accessibility, but not empty